### PR TITLE
Added german translations

### PR DESCRIPTION
**Pull request by Aquitano**
+ added de_DE.json
+ registered german locale

### CODE REVIEW

File: frontend/app_flowy/assets/translations/de_DE.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+    "appName": "AppFlowy",
+    "defaultUsername": "Ich",
+    "welcomeText": "Willkommen bei @:appName",
+    "githubStarText": "Star on GitHub",
```
Comment (lguenth) [inline frontend/app_flowy/assets/translations/de_DE.json:5]: GitHub Star vergeben

File: frontend/app_flowy/assets/translations/de_DE.json [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    "subscribeNewsletterText": "Abonniere den Newsletter",
+    "letsGoButtonText": "Lass Uns Anfangen",
```
Comment (lguenth) [inline frontend/app_flowy/assets/translations/de_DE.json:7]: Los geht's

File: frontend/app_flowy/assets/translations/de_DE.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    "title": "Titel",
+    "signUp": {
+      "buttonText": "Registrieren",
+      "title": "Registriere dich bei @:appName",
+      "getStartedText": "Los geht's",
```
Comment (lguenth) [inline frontend/app_flowy/assets/translations/de_DE.json:12]: Erste Schritte

File: frontend/app_flowy/assets/translations/de_DE.json [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+      "emptyPasswordError": "Passwort darf nicht leer sein",
+      "repeatPasswordEmptyError": "Passwortwiederholung darf nicht leer sein",
+      "unmatchedPasswordError": "Passwörter stimmen nicht überein",
+      "alreadyHaveAnAccount": "Bereits registriert?",
+      "emailHint": "E-Mail",
+      "passwordHint": "Passwort",
+      "repeatPasswordHint": "Wiederhole Passwort"
+    },
+    "signIn": {
+      "loginTitle": "Login to @:appName",
```
Comment (lguenth) [inline frontend/app_flowy/assets/translations/de_DE.json:22]: Bei @:appName einloggen

File: frontend/app_flowy/assets/translations/de_DE.json [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+      "err": {
+        "failedTitle": "Keine Verbindung zu Ihrem Konto möglich.",
+        "failedMsg": "Bitte vergewissern Sie sich, dass Sie den Anmeldevorgang in Ihrem Browser abgeschlossen haben."
+      },
+      "google": {
+        "title": "GOOGLE ANMELDUNG",
+        "instruction1": "Um Ihre Google-Kontakte zu importieren, müssen Sie diese Anwendung über Ihren Webbrowser autorisieren.",
+        "instruction2": "Kopieren Sie diesen Code in Ihre Zwischenablage, indem Sie auf das Symbol klicken oder den Text auswählen:",
+        "instruction3": "Rufen Sie den folgenden Link in Ihrem Webbrowser auf, und geben Sie den obigen Code ein:",
+        "instruction4": "Drücken Sie auf die Knopf unten, wenn Sie die Anmeldung abgeschlossen haben:"
```
Comment (lguenth) [inline frontend/app_flowy/assets/translations/de_DE.json:130]: Klicken Sie unten auf die Schaltfläche, wenn Sie die Anmeldung abgeschlossen haben:

**Review by lguenth — CHANGES_REQUESTED** [review]
Hey, thanks for the translations! I was about to fork the repo when I saw your PR.
I just added some alternatives which might be more fitting. You also left lines 5 "githubStarText" and 22 "loginTitle" untranslated, so I made suggestions. :)
Hope this helps. :grinning:

**Comment by Aquitano** [comment]
Updated PR according to the suggestions.

**Review by MikeWallaceDev — CHANGES_REQUESTED** [review]
Could you please fix translation file name, it is in the wrong format. It should be "de-DE.json"  (dash, not underscore).

**Comment by MikeWallaceDev** [comment]
@Aquitano , I'm looking at your PR now.  Did you disable the "allow edits and access to secrets by maintainers" button?  I am trying to push a small change and it's not working.

image

**Comment by Aquitano** [comment]
@MikeWallaceDev. "Allow edits ... by maintainers" is checked/enabled
Screenshot 2022-02-05 102729
.

**Comment by annieappflowy** [comment]
This PR has been pending for a while. Should we convert it into a draft or should we merge the changes?

**Pull request merged**